### ISSUE TITLE
Refine/POI fails to parse HYPERLINK() in generated XLS

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#380]
```
What steps will reproduce the problem?
1. Create a project from hyperlink_python.xls

What is the expected output? What do you see instead?

Would expect to see a single cell project with "Refine" as its text. Instead 
end up with a zero celled project. (Other times I've seen blank cells)

What version of Google Refine are you using?

2.0 & trunk.

The program that generated the .xls I've included too. As comparison, an XLS 
produced by Excel 2001 for Mac has the same (observed) data but loads OK in 
Refine.

(I'm not sure how much this is Refine's "fault" but figured I'd log it in case 
anyone else runs into this.)
```

Original issue reported on code.google.com by `paulm%pa...@gtempaccount.com` on 16 May 2011 at 1:07

Attachments:
- [hyperlink_excel2001.xls](https://storage.googleapis.com/google-code-attachments/google-refine/issue-380/comment-0/hyperlink_excel2001.xls)
- [hyperlink_python.xls](https://storage.googleapis.com/google-code-attachments/google-refine/issue-380/comment-0/hyperlink_python.xls)
- [hyperlink.py](https://storage.googleapis.com/google-code-attachments/google-refine/issue-380/comment-0/hyperlink.py)

**Comment by GoogleCodeExporter** [comment]
```
Still an issue after upgrade to Apache POI 3.7
```

Original comment by `tfmorris` on 7 Jun 2011 at 6:49

**Comment by GoogleCodeExporter** [comment]
```
My understanding is that this is caused by python-excel's xlwt not writing out 
a cached formula value. Excel will perform a formula evaluation when it loads 
the worksheet so this isn't an issue normally, but for situations like this 
where only the cached value is used it'll show up as a blank.

It's possible to either special case this in Refine or have POI do that formula 
evaluation. Ongoing conversation on python-excel,

http://groups.google.com/group/python-excel/browse_thread/thread/162ff7517d660e6
8?pli=1 (I'm waiting on the sample UNCALC file)
```

Original comment by `paulm%pa...@gtempaccount.com` on 7 Jun 2011 at 9:42

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the update.  Please let us know how the discussion turns out.  It 
it's an xlwt bug, we'll close this.  If it's a POI bug, we'll keep it open to 
track the new version of POI with the fix.
```

Original comment by `tfmorris` on 7 Jun 2011 at 4:08

**Comment by GoogleCodeExporter** [comment]
```
Any update on the investigation?  POI is about to close out their 3.8 release.
```

Original comment by `tfmorris` on 8 Mar 2012 at 3:05

**Comment by GoogleCodeExporter** [comment]
```
It sounds like the POI and python-excel teams are on this.  If and when they 
come up with a solution, Refine will pick it up as part of the next POI update, 
so I don't think we need to track this explicitly.  Closing.
```

Original comment by `tfmorris` on 18 Sep 2012 at 7:56
- Changed state: **WontFix**
